app/routers/world_clock.py

```diff
@@ -0,0 +1,744 @@
+import dateutil.parser
+import requests
+
+from datetime import datetime, timedelta
+
+from typing import Dict, List, Optional, Tuple
+
+
+TOTAL_SEC_IN_HOUR = 3600
+BLANK = ' '
+UNDERSCORE = '_'
+UNPACK_ELEMENT = 0
+TIMEZONES_BASE_URL = 'http://worldtimeapi.org/api/timezone'
+COUNTRY_TO_CONTINENT_DATA_SET = "https://pkgstore.datahub.io/JohnSnowLabs/" \
+                                "country-and-continent-codes-list/" \
+                                "country-and-continent-codes-list-csv_json/" \
+                                "data/c218eebbf2f8545f3db9051ac893d69c/" \
+                                "country-and-continent-codes-list-" \
+                                "csv_json.json"
+CITY_TO_COUNTRY_DATA_SET = "https://pkgstore.datahub.io/core/world-cities/" \
+                           "world-cities_json/data/" \
+                           "5b3dd46ad10990bca47b04b4739a02ba/" \
+                           "world-cities_json.json"
+TIMEZONES_COUNTRY_SUBCOUNTRY = {
```

Comment:
We shouldn't manage our own database of timezones. Please use `pytz.all_timezones` or something familiar :)

Reply:
I have looked into several modules in Python, including pytz, and I can not see how they help here. I have invested a lot of time and energy in writing a code that supports large amount(!) of cities and places and not just the "timezone places" themselves.
the "Mini DB" I've created is not just a list of timezones - it maps timezones to countries and subcountries and crucial for the core of my code - Ability to search the equivalent time of a custom city or place.